README.md

```diff
@@ -498,10 +502,15 @@ To enable gdb stub debugging for a project you need to enable it first in the sd
 Expand the `Component Config` section and select `ESP System Settings`. From the settings on the right for `Panic Handler behaviour` select the `GDBStub on Panic option` from the list
 ![](docs/images/GDBStubDebugging/sdkconfig_editor_panic_behavior.png)
 
-Now you will be taken to the gdbstub debugger automatically when you connect the serial monitor and there is a panic for this example create a template `hello_world` project and add the following lines in the main c file.
 
+Now you will be taken to the gdbstub debugger automatically when you connect the serial monitor and there is a panic for this example. 
+
+Create a template `hello_world` project and add the following lines in the main c file.
+
+```
 This is a global variable<br/>
-`COREDUMP_DRAM_ATTR uint8_t global_var;`
+COREDUMP_DRAM_ATTR uint8_t global_var;
+```
```

Comment:
<br/> is also appears it was part of the code block. We can change as following.


This is a global variable<br/>
```
COREDUMP_DRAM_ATTR uint8_t global_var;
```